openldap/osfamilymap.yaml

```diff
@@ -46,8 +45,6 @@ RedHat:
   slapd_pidfile: /var/run/openldap/slapd.pid
   slapd_directory: /var/lib/ldap
   slapd_modulepath: /usr/lib64/openldap
-  slapd_modules:
-    - back_mdb
```

Comment:
But where is this set other than this file?

Reply:
Check this portion on the template `slapd.conf`:
```
# Load dynamic backend modules:
modulepath	{{ openldap.slapd_modulepath }}
{%- for module in openldap.slapd_modules %}
moduleload      {{ module }}
{%- else %}
# moduleload	back_mdb
# moduleload	back_ldap
{%- endfor %}

# Load module for database
moduleload back_{{ openldap.database }}
```
Now the defaults are correctly merged, we get a duplicate line `module back_mdb` everywhere this module line is present, we can remove them

Reply:
Maybe I need to actually run the formula to see the problem... this is where having Kitchen + Travis set up makes a big difference.